### PR TITLE
Add support for Nicaragua provider - v2

### PR DESCRIPTION
**Pull request by ialokim**
Please add support and logo for the newly on navitia.io integrated Nicaragua coverage.

I've rebuild #389 with the new navitia.io coverage and the current master of the staging PTE, having only one provider for whole Nicaragua, I've added a new region called "Central America", too.

I've compiled the app and it already works on my Android device.

See #418 for the v1 version of this PR.

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -204,8 +204,6 @@ dependencyVerification {
 			'com.github.tony19:apktool-lib:1.4.4-3:apktool-lib-1.4.4-3.jar:bdc4e93b1dd6661698feedacc37d203801076deddd7fb350daf98611637a5cd6',
 			'com.github.tony19:logback-android-classic:1.1.1-6:logback-android-classic-1.1.1-6.jar:0d0939c2cd797ecb0b7d679f1e2314a9889a90a633f2fc5b66f11316cbd6626b',
 			'com.github.tony19:logback-android-core:1.1.1-6:logback-android-core-1.1.1-6.jar:9548fd28988d62a8f67b72fafc67167f91f061001513ae25f53921f2982c8c01',
-			'com.gitlab.opentransitmap.public-transport-enabler:enabler:f540e9748e:enabler-f540e9748e.jar:75aa66b301ce38c7447cba8c982751bf0c789b492ee186f7bc0d47899dfb7c73',
-			'com.gitlab.opentransitmap:public-transport-enabler:f540e9748e:public-transport-enabler-f540e9748e.jar:2b947050d2a6112366aa5a349169bdd1c43584e56d2faf411f9de9617086e58b',
```
Comment (grote) [inline app/build.gradle]: This should be updated instead of removed. You can get the checksums by running `gradle -q calculateChecksums`

**Comment by grote** [comment]
@cla-bot[bot] check

File: app/build.gradle (hunk 1 above)
Comment (ialokim) [reply app/build.gradle]: Done 👍

**Comment by ialokim** [comment]
I've pushed the updated version of this PR, not sure why Travis fails, seems to be some problem with the AVD.

**Comment by xamanu** [comment]
This is wonderful! Thanks to everybody involved to bring the Nicaragua providers into Transportr!!

**Comment by grote** [comment]
One translator opened this issue about the new string introduced here:

https://www.transifex.com/grobox/transportr/translate/#fr/strings/135526775/

**Pull request merged**